src/modules/returns-notifications/lib/returns-notification-send.js

```diff
@@ -0,0 +1,39 @@
+const { logger } = require('../../../logger');
+const send = require('../lib/send');
+const notify = require('../../notify');
+
+const JOB_NAME = 'returnsNotification.send';
+
+/**
+ * Creates a message for Bull MQ
+ * @param {Object} data containing message data options
+ * @returns {Object}
+ */
+const createMessage = data => {
+  logger.info(`Create Message ${JOB_NAME}`);
+  return [
+    JOB_NAME,
+    data,
+    {
+      jobId: `${JOB_NAME}`
+    }
+  ];
+};
+
+const handleReturnsNotificationSend = async job => {
+  logger.info(`Handling: ${JOB_NAME}:${job.id}`);
+};
+
+const onComplete = async (job, queueManager) => {
+  const options = await send.prepareMessageData(job.data);
```

Comment:
Why did you not add these steps in the handler? Is it not better to perform the job functions in the handler rather than complete?